packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -29,13 +32,26 @@ const emit = defineEmits<{
    * Emitted when the current search input value changes.
    */
   "update:searchTerm": [searchTerm: string];
+  "update:open": [open: boolean];
   /**
    * Emitted if lazy loading is triggered / the users scrolls to the end of the options.
    * See property `lazyLoading` for enabling the lazy loading.
    */
   lazyLoad: [];
 }>();
 
+const searchTerm = useManagedState(
```

Comment:
Managing the search term internally would only make sense if we also implement the filter logic, right?

Reply:
I wouldn't think so. It should not have any additional side-effects in my opinion

Reply:
Sure it has no side effects but imho if its not used anywhere we don't need to add it :D
Otherwise its a little useless / unnecessary complexity

Reply:
Well, I would say it still has the benefit of of the user not needing to care about caching the searchterm:
```vue
<OnyxSelect @update:searchTerm="val => options = all.filter(...)"
```
This way the component still works as expected.